theme/fontsize.css

```diff
@@ -3,18 +3,24 @@
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
  */
 
-.text-tiny {
-	font-size: .7em;
-}
+/* The values should be synchronized with the "FONT_SIZE_PRESET_UNITS" object in the "/src/fontsize/utils.js" file. */
```

Comment:
This line was removed in 2f33bedced933ca908efe351f178c3db42bb49c9 and as I can see, it appeared again :D

Reply:
If it is unnecessary, please get rid of it on master.